src/backend/InvenTree/generic/states/fields.py

```diff
@@ -138,14 +143,21 @@ def add_field(self, cls, name):
         if self.status_class:
             validators.append(CustomStatusCodeValidator(status_class=self.status_class))
 
+        extra_help_text = (
+            '. Can be any of the keys in the corresponding status field, as well as any defined custom keys for the status type.'
+            if InvenTree.ready.isGeneratingSchema()
+            else ''
+        )
```

Comment:
The python ternary operator feels awkward to me, is there a better way this should have been written?